priv/repo/migrations/20220526100943_add_keyword_field.exs

```diff
@@ -0,0 +1,9 @@
+defmodule GoogleScraping.Repo.Migrations.AddKeywordField do
```

Comment:
The migration file name should be `...add_html_to_users` based on out convention.

> [ChangeMade]+To/From+[TableNamePlural]

We follow this convention

Reply:
Noted. Migration was renamed: https://github.com/Nihisil/nimble-certification-elixir/pull/41/commits/6b99c136afaf67d9752af2a157c2b938c22336b9